src/sourmash/commands.py

```diff
@@ -350,7 +350,7 @@ def index(args):
 
     if args.sparseness < 0 or args.sparseness > 1.0:
         error('sparseness must be in range [0.0, 1.0].')
-
+    
```

Comment:
```suggestion

```
unnecessary space added, probably unintentionally?

Reply:
yes, i'll remove it